app/models/katello/concerns/redhat_extensions.rb

```diff
@@ -44,7 +44,12 @@ def kickstart_repos(host)
         distros = distribution_repositories(host)
         if distros && host.content_source
           distros.map do |distro|
-            {:id => distro.id, :name => distro.name, :path => distro.full_path(host.content_source)}
+            {
+              id: distro.id,
+              name: distro.name,
+              label: distro.label,
+              path: distro.full_path(host.content_source)
+            }
```

Comment:
I should point out this conflicts with changes in https://github.com/Katello/katello/pull/7774.  Not a big deal, just thought I'd mention it since whichever one gets merged second will need to make some changes.

Reply:
Thanks. I'll rebase once yours go through